packages/projection-typeorm/src/entity/Handle.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { AssetEntity } from './Asset.entity';
+import { Cardano } from '@cardano-sdk/core';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryColumn } from 'typeorm';
+
+@Entity()
+export class HandleEntity {
+  @PrimaryColumn()
+  handle?: string;
```

Comment:
All the properties of other entities have this "?" operator. @mkazlauskas should be able to say the reason for this.

Reply:
This is resolved but it's not clear to me (the reviewer) why

Reply:
Thanks for the explanation @iccicci. To summarise: the types are nullable since the selection could be made without including all fields in the query.